src/status_im2/setup/config.cljs

```diff
@@ -142,3 +142,5 @@
 ;; TODO: Remove this (highly) temporary flag once the new Activity Center is
 ;; usable enough to replace the old one **in the new UI**.
 (def new-activity-center-enabled? true)
+
+(def local-pairing-mode-enabled? (atom false))
```

Comment:
could you elaborate on why this is needed?

Reply:
Got rid of this and now using env vars config for this feature